UrCan/Blok 2 - Tools/GeneratorEditor.cs

```diff
@@ -0,0 +1,75 @@
+﻿using UnityEngine;
+using UnityEditor;
+using System.IO;
+using System.Collections.Generic;
+[CustomEditor(typeof(Generator))]
+public class GeneratorEditor : Editor {
+	private int mode;
+
+	public static string FILE_TYPE = "bruut";
```

Comment:
Brute code man, echt sick.
Zelfs een screenshot erbij, elke vraag die ik wilde stellen heb je beantwoord met wat er in de files staat. Top werk.